app/engine/engine.rb

```diff
@@ -107,7 +107,7 @@ def f(tms)
     def output(players, benchmarks)
       turns = players.map.with_index do |player|
         value = "%4s" % player.handler.value
-        "#{player.robot.username}: #{value}"
+        "#{player.robot.username}: move #{value} armor #{player.robot.armor}"
```

Comment:
This made it easier to debug what was going on - since it made clear where robots were getting shot. Seemed useful to keep around